frame/staking/src/pallet/impls.rs

```diff
@@ -103,7 +119,8 @@ impl<T: Config> Pallet<T> {
 		controller: &T::AccountId,
 		num_slashing_spans: u32,
 	) -> Result<Weight, DispatchError> {
-		let mut ledger = Self::ledger(&controller).ok_or(Error::<T>::NotController)?;
+		let mut ledger =
+			Self::ledger(Controller(controller.clone())).ok_or(Error::<T>::NotController)?;
```

Comment:
I updated the PR to make the code more consistent but instead of removing `Self::ledger()`, I replaced all the `StakingLedger` getters by `Self::ledger`. This way I think the code looks more readable and less cluttered, whilst consistent.